app/views/layouts/admin/_admin_navigation.html.erb

```diff
@@ -16,6 +16,13 @@
             <li><%= link_to 'Get Started', new_user_registration_path, class: 'btn btn-outline-primary px-3 py-2 rounded-0' %></li>
           <% end %>
         </ul>
-      </ul>
+      </div>
+    </nav>
+
+    <div class="row py-2">
+      <h3><%= controller_name.titleize %></h3>
+
+      <%# <%= link_to 'Add Survey', polymorphic_path([:new,:admin, controller_name.singularize.to_sym]) %1> %>
```

Comment:
you could have `link_to 'Add Survey', [:new,:admin, controller_name.singularize.to_sym]` where by default `url_for` is applied to  second argument